### ISSUE TITLE
Custom receipt has wrong background color

### ISSUE THREAD

**Issue by bjosttveit** [Altinn/app-frontend-react#2551]
### Description of the bug

The custom receipt seems to get a gray background except from the app-bar which looks very strange compared to the standard receipt.

Custom receipt:
![Image](https://github.com/user-attachments/assets/80043d7c-e49e-4e30-971f-83570bc36540)

Standard receipt:
![Image](https://github.com/user-attachments/assets/b753ad38-8b06-499c-a13b-517d2d2dbc3d)

### Steps To Reproduce

Add a custom receipt https://docs.altinn.studio//nb/altinn-studio/reference/configuration/process/customize/#custom-form-layout-1

**Comment by olemartinorg** [comment]
Tested in main now, and it works as expected. I suspect it has been working for a while, as I can't remember seeing this issue lately.

**Comment by bjosttveit** [comment]
It was very flaky, so I'm not sure

**Comment by bjosttveit** [comment]
I will see if I am able to reproduce this again, but I suspect its not optimal given that we still do it like this: 

<img width="889" alt="Image" src="https://github.com/user-attachments/assets/4165345e-4649-460e-a25b-9a5f1072ced7" />

**Comment by olemartinorg** [comment]
Yep, I think we should use `react-helmet` or something for this instead.. Fix it using a tried-and-tested method, and assume it won't be as flaky in the future.
